diff --git a/dist/transformers.node.mjs b/dist/transformers.node.mjs
index 2289efb..23adb2b 100644
--- a/dist/transformers.node.mjs
+++ b/dist/transformers.node.mjs
@@ -1,6 +1,9 @@
 import * as __WEBPACK_EXTERNAL_MODULE_fs__ from "fs";
 import * as __WEBPACK_EXTERNAL_MODULE_onnxruntime_common_82b39e9f__ from "onnxruntime-common";
-import * as __WEBPACK_EXTERNAL_MODULE_onnxruntime_node_6a60201e__ from "onnxruntime-node";
+let __WEBPACK_EXTERNAL_MODULE_onnxruntime_node_6a60201e__;
+if (!(Symbol.for('onnxruntime') in globalThis)) {
+    __WEBPACK_EXTERNAL_MODULE_onnxruntime_node_6a60201e__ = await import("onnxruntime-node");
+}
 import * as __WEBPACK_EXTERNAL_MODULE_path__ from "path";
 import * as __WEBPACK_EXTERNAL_MODULE_sharp__ from "sharp";
 import * as __WEBPACK_EXTERNAL_MODULE_url__ from "url";
@@ -2330,3 +2333,8 @@
 if (ORT_SYMBOL in globalThis) {
     // If the JS runtime exposes their own ONNX runtime, use it
     ONNX = globalThis[ORT_SYMBOL];
+
+    if (ONNX?.env?.wasm) {
+        supportedDevices.push('wasm');
+        defaultDevices = ['wasm'];
+    }
